book/integration-service.md

```diff
@@ -0,0 +1,271 @@
+# Integration Service
+
+
+## Overview
+
+The Integration Service is composed of a couple controllers that facilitate automated testing of content produced by the HACBS build pipelines. It is responsible for integration testing capabilities.
+
+To support this capability, it needs to do a couple of things.
+
+- When a build pipeline completes, it creates an Snapshot CR representing a new collection of components that should be tested together.
+- When it sees a new Snapshot CR (created either by itself or by a user), it coordinates deployment of the application for testing. It does this by creating a new “ephemeral” StoneSoup Environment CRs which should in turn trigger the GitOps Service to provision environment (and a sub-workspace in CPS)
```

Comment:
How many CRs are created? If one, change to `StoneSoup Environment CR`. If multiple, change to `creating new "ephemeral"`.

Remove `should in turn`.

What is `CPS`? I do not see it defined.

Reply:
This is a common issue in our docs unfortunately, since initially it was supposed to be only one CR. Based on the current design, multiple CRs will be created, so I agree with the proposal to drop the `a` from the `a new "ephemeral"` part.

Reply:
The `(and a sub-workspace in CPS)` part can be removed entirely. It was referring to the KCP-related part of the design and I overlooked it when refactoring.

Reply:
It is an annoying balance to try to maintain between being specific enough for clarity but not too specific so that your documentation gets outdated with the slightest changes...